$ CFLAGS="... -fno-strict-aliasing" cmake . && cmake --build .
/usr/bin/cc  ... -fno-strict-aliasing;-std=gnu11 ...
cc: error: no input files
/bin/sh: -std=gnu11: not found

--- CMakeLists.txt.orig	2024-07-10 23:57:57 UTC
+++ CMakeLists.txt
@@ -257,7 +257,7 @@ endif()
 		-Wno-macro-redefined
 	)
 endif()
-list(APPEND CMAKE_C_FLAGS -std=gnu11)
+set(CMAKE_C_STANDARD 11)
 
 if (CMAKE_BUILD_TYPE STREQUAL "Profile")
 	amsg("${CL_WHT}------------------*Profile Build*------------------${CL_RST}")
